packages/cli/src/create.ts

```diff
@@ -125,18 +125,26 @@ export async function create(): Promise<void> {
 }
 
 async function promptForResourceCreation(ccfg: NewRenderedClusterConfigType) {
+  const tos = "https://go.opstrace.com/terms-of-service";
```

Comment:
`"go/terms-of-service" was not found`

Reply:
Yes, I converted to draft until we have the text.